src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/ResidentialChecksTaskAnswer.kt

```diff
@@ -0,0 +1,66 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.DiscriminatorColumn
+import jakarta.persistence.DiscriminatorType
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Inheritance
+import jakarta.persistence.InheritanceType
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotNull
+import java.time.LocalDateTime
+
+enum class ResidentialChecksTaskAnswerType {
+  ADDRESS_DETAILS_AND_INFORMED_CONSENT,
+  POLICE_CHECK,
+  CHILDREN_SERVICES_CHECK,
+  ASSESS_THIS_PERSONS_RISK,
+  SUITABILITY_DECISION,
+  MAKE_A_RISK_MANAGEMENT_DECISION,
+}
+
+@Entity
+@Table(name = "residential_checks_task_answer")
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@DiscriminatorColumn(name = "answer_type", discriminatorType = DiscriminatorType.STRING)
+abstract class ResidentialChecksTaskAnswer(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
might be nice to sthis aubclasses in their own subpackage? (if so it could be a follow up)

Reply:
yes there's seven related classes so a separate package would make sense